autoscale/src/test/java/com/sequenceiq/periscope/modul/rejected/RejectedThreadContext.java

```diff
@@ -59,6 +60,7 @@
     @MockBean({Clock.class, ClusterService.class, CloudbreakClientConfiguration.class,
             MetricUtils.class, InternalCrnBuilder.class, FailedNodeRepository.class})
     @EnableAsync
+    @Profile("dev")
```

Comment:
Accidental change? or should this be test. Also other places where this change is made.

Reply:
This is correct, this context is used to setup specific thread pool size for dev mode testing.